go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/onsi/gomega v1.18.1
 	github.com/operator-framework/api v0.13.0
 	github.com/operator-framework/helm-operator-plugins v0.0.9
-	github.com/spf13/cobra v1.3.0
+	github.com/spf13/cobra v1.4.0
```

Comment:
nit: I wouldn't expect the base go.mod to have a diff if we are only adding a tool, which has a separate hacks/tools/go.mod